keyboards/mokey/mokey64/info.json

```diff
@@ -0,0 +1,82 @@
+{
+    "keyboard_name": "Mokey64",
+    "url": "",
+    "maintainer": "mokey",
+    "width": 14,
+    "height": 5,
```

Comment:
Please just commit the suggestion.

Reply:
Hello, I submitted to qmk for the first time. I'm not familiar with the process and want to understand it. Now info.json contains "width": 14 and "height": 5. Just submitted many times, it's the same error prompt. Can you tell me what to do next? Do you need to continue modifying in info.json or "submit" on GitHub?Hello, I submitted to qmk for the first time. I'm not familiar with the process and want to understand it. Now info.json contains "width": 14 and "height": 5. Just submitted many times, it's the same error prompt. Can you tell me what to do next? Do you need to continue modifying in info.json or "submit" on GitHub?

Reply:
Hello, I submitted to qmk for the first time. I'm not familiar with the process and want to understand it. Now info.json contains "width": 14 and "height": 5. Just submitted many times, it's the same error prompt. Can you tell me what to do next? Do you need to continue modifying in info.json or "submit" on GitHub?